build/Helix/PrepareHelixPayload.ps1

```diff
@@ -0,0 +1,54 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform,
+    [string]$Configuration,
+    [string]$ArtifactName='drop'
+)
+
+$payloadDir = "HelixPayload\$Configuration\$Platform"
+
+$repoDirectory = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "..\..\"
+$nugetPackagesDir = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "packages"
+ 
+# Create the payload directory. Remove it if it already exists.
+If(test-path $payloadDir)
+{
+    Remove-Item $payloadDir -Recurse
+}
+New-Item -ItemType Directory -Force -Path $payloadDir
+
+# Copy files from nuget packages
+Copy-Item "$nugetPackagesDir\microsoft.windows.apps.test.1.0.181203002\lib\netcoreapp2.1\*.dll" $payloadDir
+Copy-Item "$nugetPackagesDir\taef.redist.wlk.10.57.200731005-develop\build\Binaries\$Platform\*" $payloadDir
+Copy-Item "$nugetPackagesDir\taef.redist.wlk.10.57.200731005-develop\build\Binaries\$Platform\CoreClr\*" $payloadDir
+New-Item -ItemType Directory -Force -Path "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.2.1.0\runtimes\win-$Platform\lib\netcoreapp2.1\*" "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.2.1.0\runtimes\win-$Platform\native\*" "$payloadDir\.NETCoreApp2.1\"
+
+function Copy-If-Exists
+{
+    Param($source, $destinationDir)
+
+    if (Test-Path $source)
+    {
+        Write-Host "Copy from '$source' to '$destinationDir'"
+        Copy-Item -Force $source $destinationDir
+    }
+    else
+    {
+        Write-Host "'$source' does not exist."
+    }
+}
```

Comment:
Wait is this used?

Reply:
No, well, it was and now it isn't. But I want to leave it here to reserve the right to use it again as we add more artifacts to Helix that may or may not exist on scenario or build flavor.